api/src/pcapi/routes/adage/v1/serialization/educational_deposit.py

```diff
@@ -16,17 +21,17 @@ class Config:
 
 
 def serialize_educational_deposits(
-    educational_deposits: list[EducationalDepositNamedTuple],
+    educational_deposits: list["EducationalDeposit"],
 ) -> list[EducationalDepositResponse]:
     serialized_educational_deposit = []
     for educational_deposit in educational_deposits:
         serialized_educational_deposit.append(serialize_educational_deposit(educational_deposit))
     return serialized_educational_deposit
```

Comment:
(style) On pourrait faire plus concis avec une list comprehension : 

```suggestion
    return [serialize_educational_deposit(d) for d in educational_deposits]
```

Reply:
Raphael n'est pas fan des comprehension de list, me concernant tant que je fais plaisir :) 
Par contre hors de question de nommer ma variable d, 
Voici ma modification : 
[serialize_educational_deposit(educational_deposit) for educational_deposit in educational_deposits]

Reply:
Je n'aime pas les list comprehensions quand elles sont trop complexes. Quand elles tiennent sur une ligne, ça me va, perso. ;)

(Et avoir un nom de variable très court comme `d` ou `deposit` dans une list comprehension aussi courte reste assez lisible, à mon goût.)